Training25/Program.cs

```diff
@@ -4,10 +4,28 @@
 // ------------------------------------------------------------------
 // Program.cs
 // Program on main branch.
+// T12: Program to find winner of voting contest.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
 namespace Training25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      do {
+         Clear ();
+         Write ("Enter input: ");
+         var input = ReadLine ();
+         if (!string.IsNullOrEmpty (input) && input.ToLower ().All (x => x >= 'a' && x <= 'z'))

```

Comment:
```suggestion
        if (!string.IsNullOrEmpty (input) && input.All (char.IsLetter))

```
No need to use `ToLower()` again here as it was handled in the `FindPerson ()` method